### PR TITLE
Update release details for 1.22.2

### PR DESCRIPTION
**Pull request by keirthana**
Release updates for 1.22.2, pending tasks:

- [x] Verify component versions
- [x] Add release notes link once announcement is created

### CODE REVIEW

**Review by ajanon — APPROVED** [review]
LGTM

File: reference/component-versions.md [hunk 1]
```diff
@@ -2,6 +2,40 @@ This documents the versions of the different components for each Anbox Cloud rel
...
+
+| Name |  Channel | Version |
+|----------|--------------|---------|
+| `ams`    | `1.22/stable` | 1.22.2 |
+| `aar`    | `1.22/stable` | 1.22.2 |
+| `ams-node-controller` | `1.22/stable` | 1.22.2 |
+| `anbox-cloud-dashboard` | `1.22/stable` | 1.22.2 |
+| `anbox-stream-agent` | `1.22/stable` | 1.22.2 |
+| `anbox-stream-gateway` | `1.22/stable` | 1.22.2 |
+| `anbox-cloud-appliance` | `1.22/stable` | 1.22.2 |
```
Comment (adglkh) [inline reference/component-versions.md:25]: As discussed in SU, the anbox-cloud-appliance would remain 1.22.1 as there is no update to the snap package.

**Review by adglkh — CHANGES_REQUESTED** [review]

File: reference/component-versions.md [hunk 2]
```diff
@@ -2,6 +2,40 @@ This documents the versions of the different components for each Anbox Cloud rel
...
+
+| Name |  Channel | Version |
+|----------|--------------|---------|
+| `ams`    | `1.22/stable` | 1.22.2 |
+| `aar`    | `1.22/stable` | 1.22.2 |
+| `ams-node-controller` | `1.22/stable` | 1.22.2 |
+| `anbox-cloud-dashboard` | `1.22/stable` | 1.22.2 |
+| `anbox-stream-agent` | `1.22/stable` | 1.22.2 |
+| `anbox-stream-gateway` | `1.22/stable` | 1.22.2 |
+| `anbox-cloud-appliance` | `1.22/stable` | 1.22.1 |
```
Comment (adglkh) [inline reference/component-versions.md:25]: One small question: do we need to list the snap like anbox-cloud-appliance here if there is no update for it in one patch release?

Comment (keirthana) [reply reference/component-versions.md:25]: Wouldn't it be useful to know which version of snap we are on? I don't mind removing it either.

**Pull request merged**